pages/installing-slimefun.md

```diff
@@ -0,0 +1,45 @@
+# Installing Slimefun
+
+## Prerequisites
+* A 1.14.4 Spigot Minecraft server.
+* The latest version of Slimefun 4, which can be found [here](https://thebusybiscuit.github.io/builds/TheBusyBiscuit/Slimefun4/master/).
+* Slimefun 4 will utilize [CS-CoreLib](https://thebusybiscuit.github.io/builds/TheBusyBiscuit/CS-CoreLib/master/) as a mandatory dependency.
```

Comment:
I think "utilize" is the wrong word here, you should maybe just outright tell them they will also need CS-CoreLib.